Sources/PointFree/Account/Account.swift

```diff
@@ -481,7 +481,10 @@ private let subscriptionPaymentInfoView = View<Stripe.Subscription> { subscripti
 }
 
 private func totalAmount(for subscription: Stripe.Subscription) -> String? {
-  let totalCents = subscription.plan.amount.rawValue * subscription.quantity
+  let totalCents = subscription.plan.amount.rawValue
```

Comment:
Might wanna pull the calculation into `Stripe.Subscription`. Also not sure if the rounding rules are right yet...